test/specs/lazyload.spec.js

```diff
@@ -111,8 +111,8 @@ describe('LazyLoad', () => {
           </div>
         , div);
 
-        // Differnce between the test above
-        expect(document.querySelector('.test1 .times').textContent).to.equal('3');
+        // Difference between the test above
+        expect(document.querySelector('.test1 .times').textContent).to.equal('2');
```

Comment:
I'm trying to change this to spy the `checkVisible` calls, as it makes more sense.